src/main/antlr4/imports/mysql_idents.g4

```diff
@@ -38,13 +38,13 @@ SQL_COMMENT: '/*' ~'!' (.)*? '*/' -> skip;
 
 SQL_LINE_COMMENT: ('#' | '--') (~'\n')* ('\n' | EOF) -> skip;
 
-STRING_LITERAL: TICK ('\\\'' | '\'\'' | ~('\''))* TICK;
+STRING_LITERAL: [a-zA-Z]? TICK ('\\\'' | '\'\'' | ~('\''))* TICK;
```

Comment:
it's actually not liberal enough, https://dev.mysql.com/doc/refman/5.7/en/literals.html shows there's some stuff I'm still missing (`_utf8'string'`).  But ok, I'll see what's available.